tests/astar_test.py

```diff
@@ -0,0 +1,23 @@
+import unittest
+import numpy
+from worldengine import astar
+
+
+class TestCommon(unittest.TestCase):
+
+    def test_traversal(self):
```

Comment:
nice test. Related thing: it could be useful to have some documentation to understand how astar should be used. For examples, does 0.0 represents a point that cannot be traversed or it is something that can be traversed for free?